### PR TITLE
update local-bmo.sh to work with cluster-baremetal-operator

### PR DESCRIPTION
**Pull request by dhellmann**

### CODE REVIEW

File: stop-cbo.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+source common.sh
+
+# Save the image configuration in case the goal is to start a local
+# copy of the CBO. If there is no pod, we don't want the command to
+# fail.
+pod=$(oc get pod -n openshift-machine-api -o name | grep cluster-baremetal-operator || true)
+if [ -n "$pod" ]; then
+    oc exec $pod -n openshift-machine-api -c cluster-baremetal-operator \
+       -- cat /etc/cluster-baremetal-operator/images/images.json > ${OCP_DIR}/cbo-images.json
+    echo "CBO image settings saved to ${OCP_DIR}/mao-images.json"
```
Comment (andfasano) [inline stop-cbo.sh:12]: typo: mao-images.json

Comment (dhellmann) [reply stop-cbo.sh]: Thanks, fixed.

**Pull request merged**